.github/workflows/test-automation.yml

```
@@ -59,11 +70,37 @@
         run: npm run test:integration
         env:
           NODE_ENV: test
+          CI: true
+          METRICS_API_KEY: test-metrics-key
+          ADMIN_API_KEY: test-admin-key
+          SENTRY_DSN: https://test@sentry.io/test
+          VERCEL_URL: http://localhost:3000
+          CI: true
```

actionlint (1.7.7)

key "CI" is duplicated in env. previously defined at line:73,col:11. note that this key is case insensitive

(syntax-check)